### PR TITLE
🚀[Release v.2.10.0-20240421] Merge into Develop

### PR DESCRIPTION
**Pull request by kcw-grunt**
# Resolved in Main 

# Updates ....alot

* Removed sensitive data 
* In Progress Refactoring Code
- Removed cruft
- bump version
- working ops address being generated 
* tieredOps working
- refactor cleanup
- renaming and refactor…project running!
- created and sent a transaction
- Renamed some code from Bitcoin to Litecoin
- remove whitespace
- created a set to create the intersection
- Filtered out the address as targeted
- added textview for Service Fee
- condensed the inits
- Renamed from iso to more meaningful names 
* More refactor
- Moved Fee Manager
- Changed the rate polls
- renamed getIso to getIsoSymbol
- Removed saveEvent
- Adjusted hot spot of Amount Edit
- added full names to buttons.
- added extra check before donating
- commented out the new params
- Fixes the Unsopported ISO bug
- Removed old code 
* Relabeled Bitcoin to Litecoin in places
- Cleaned up the detail text in Transaction Item
- updated confirmation label
- Tried hide the keyboard  
- updated tiers : 1713522152 
* Debug of the com.breadwallet.tools.security.AuthManager.authPrompt
- moved google serivces plugin to the bottom per Pusher Suggest
- Added Functionality to button yes language (#178)
 - Put announce updates view on create and recreate (#175)
* Put AnnounceUpdatesViewActivity in button create and restore
* Added enable on back invoke call on Manifest
* Added src/main/assets inside fast_follow_asset_pack_01 folder
* Create Folder src/main/assets
* delete test file
* Create directory src/main/assets inside fast_follow_asset_pack_01
* Rename file to empty_file

### CODE REVIEW

File: app/build.gradle [hunk 1]
```diff
@@ -213,13 +213,13 @@ android {
     }
 
     buildTypes {
-//        gitUpdateModules()
-//        proguardFiles getDefaultProguardFile('proguard-android.txt'), 'proguard-rules.pro'
-
         release {
             signingConfig signingConfigs.release
-            minifyEnabled = false
+            minifyEnabled = true
```
Comment (josikie) [inline app/build.gradle:218]: What is minify?

Comment (josikie) [inline app/build.gradle:218]: What's minifyEnabled?

**Review by josikie — APPROVED** [review]
Cool! Thanks @kcw-grunt

File: app/build.gradle (hunk 1 above)
Comment (kcw-grunt) [reply app/build.gradle:218]: This is a feature AS does to reduce the size of the binary by compressing the libraries

**Pull request merged**